src/main/java/com/cburch/logisim/std/gates/ControlledBuffer.java

```diff
@@ -265,7 +265,7 @@ public void propagate(InstanceState state) {
     final var width = state.getAttributeValue(StdAttr.WIDTH);
     if (control == Value.TRUE) {
       final var in = state.getPortValue(1);
-      state.setPort(0, isInverter ? in.not() : in, GateAttributes.DELAY);
+      state.setPort(0, isInverter ? in.not() : in.not().not(), GateAttributes.DELAY);
```

Comment:
Please put your comment from the conversation here to mark `not().not()` is for a reason.